connect/main.go

```diff
@@ -80,7 +79,8 @@ func authenticate(_ *cobra.Command, args []string) {
 
 	fmt.Print("Password: ")
 
-	password, err := terminal.ReadPassword(syscall.Stdin)
+	//password, err := terminal.ReadPassword(syscall.Stdin)
```

Comment:
Let's remove this line - it's preserved in git history if we ever need it again.

Reply:
Ya true i just left this here for testing.